docs/data/base/components/popper/popper.md

```diff
@@ -0,0 +1,40 @@
+---
+product: base
+title: React Popper component
+components: PopperUnstyled
+githubLabel: 'component: Popper'
+---
+
+# Unstyled popper
+
+<p class="description">The Popper component lets you create components that are displayed on top of other elements.</p>
+
+**Caveats**:
+
+- 🕷 Popper relies on the 3rd party library ([Popper.js v2](https://popper.js.org/docs/v2/)) for perfect positioning.
+- 💄 It's an alternative API to [react-popper](https://popper.js.org/react-popper/v2/). It aims for simplicity.
+- Clicking away does not hide the `Popper` component. If you need this behavior, you can use [`ClickAwayListener`](/base/react-click-away-listener/)
```

Comment:
I'd say this is the only "caveat". The other bullet points should be under "features"

Reply:
Done!